src/platform/packages/shared/shared-ux/error_boundary/src/ui/message_components.tsx

```diff
@@ -34,6 +34,57 @@ interface FatalPromptProps {
   onClickRefresh: () => void;
 }
 
+const CodePanel: React.FC<CodePanelProps> = (props) => {
```

Comment:
Moved this back where it was to reduce the LOC changes from main in https://github.com/elastic/kibana/pull/204315/files (as a nit)